test/exp_test_helper.py

```diff
@@ -0,0 +1,188 @@
+
+from __future__ import print_function
```

Comment:
Were the people who wrote this code before us running the code using `python 2.x`? Are there any possible trip ups running it with a more modern version of `python`? @aidanheerdegen would you know?

Reply:
Yeah it is super old. I don't know if there are any other `python 2.x` features, but we should definitely get rid of them if there are. Don't want that crufty stuff hanging around.

Are there some linters/tools that can find that stuff to ensure it has all been eradicated?

Reply:
I used http://python3porting.com/differences.html and went through the file to confirm that that was the only porting difference.